.gitignore

```diff
@@ -1,2 +1,4 @@
+Gemfile.lock
```

Comment:
In the [lock file debate](
http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/)
my position is: treat your app as a gem for as long as possible/
reasonable. If there are issues with certain libraries on some devs
machine, we should know about it. In this case, your call.